operations/conman/ConMan.md

```diff
@@ -2,12 +2,12 @@
 
 ConMan is a tool used for connecting to remote consoles and collecting console logs. These node logs can then be used for various administrative purposes, such as troubleshooting node boot issues.
 
-ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`. ConMan can be configured to encrypt usernames and passwords. Node console logs are stored locally within the `cray-console-operator pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
+ConMan runs on the system as a containerized service. It runs in a set of Docker containers within Kubernetes pods named `cray-console-operator` and `cray-console-node`.
+Node console logs are stored locally within the `cray-console-node-0 pod` in the `/var/log/conman/` directory, as well as being collected by the System Monitoring Framework \(SMF\).
```

Comment:
Not to be picky, but should the word 'pod' be inside the ``?

Reply:
Come to think of it, may be better to say "within the `cray-console-node` pods in the `/var/log/conman/` directory.